.rubocop.yml

```diff
@@ -13,6 +14,7 @@ AllCops:
 # Metrics/LineLength:
 #   Max: 120
 
+
 Style/StringLiterals:
   Enabled: true
   EnforcedStyle: single_quotes
```

Comment:
I believe this is consistent with current practice in other projects.

I should remove that extra blank line, though.